homeassistant/components/recorder/table_managers/recorder_runs.py

```diff
@@ -58,9 +28,7 @@ def recording_start(self) -> datetime:
     @property
     def first(self) -> RecorderRuns:
         """Get the first run."""
-        if runs_by_timestamp := self._run_history.runs_by_timestamp:
-            return next(iter(runs_by_timestamp.values()))
-        return self.current
+        return self._first_run or self.current
```

Comment:
The `first` property is used for system health. Considering we're no longer relying on the recorder runs to avoid unnecessary history queries, but instead the timestamp of the oldest state, we should perhaps remove this property and include the timestamp of the oldest state in the system health information instead?

Reply:
Sounds like a good change to me

Reply:
OK, I'll do that in a follow-up 👍